deploy.cmd

```diff
@@ -89,20 +89,26 @@ goto :EOF
 echo Handling node.js deployment.
 
 
-:: 1. Select node version
+echo 1. Select node version
 call :SelectNodeVersion
 
-:: 2. Install npm packages
+echo  2. Install npm packages
 IF EXIST "%DEPLOYMENT_SOURCE%\package.json" (
   pushd "%DEPLOYMENT_SOURCE%"
-  call :ExecuteCmd !NPM_CMD! install --production
+  call :ExecuteCmd !NPM_CMD! install
   IF !ERRORLEVEL! NEQ 0 goto error
   popd
 )
-:: 3. Angular Prod Build
-IF EXIST "%DEPLOYMENT_SOURCE%/.angular.json" (
+echo 3. Angular Prod Build
+IF EXIST "%DEPLOYMENT_SOURCE%/angular.json" (
 echo Building App in %DEPLOYMENT_SOURCE%…
 pushd "%DEPLOYMENT_SOURCE%"
+cd ./node_modules/@angular/cli
+rd /s /q node_modules
```

Comment:
Is there a specific reason to remove the existing `node_modules` folder? This will slow deployment dramatically. Also curious why we have to do an `npm install` within a subfolder of the `node_modules` folder?

Reply:
See [this for removing node_modules](https://github.com/angular/angular-cli/issues/9654#issuecomment-380206565) and [ this for installing inside the @angular-cli module](https://github.com/keymanapp/donate.keyman.com/pull/7#discussion_r271346049) and

Reply:
That second link points to this repo.